codalab/rest/users.py

```diff
@@ -110,3 +117,22 @@ def fetch_users():
     usernames.discard('')  # str.split(',') will return '' on empty strings
     users = local.model.get_users(usernames=(usernames or None))
     return UserSchema(many=True).dump(users).data
+
+
+@patch('/users')
+def update_users():
+    """Update users."""
+    if request.user.user_id != local.model.root_user_id:
+        abort(httplib.FORBIDDEN, "Only root user can update users.")
+
+    # Deserialize and update
+    users = AuthenticatedUserSchema(
```

Comment:
Are we every going to have more than one user coming back here?  If not, assert that's the case and specialize code to that case.